src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -41,9 +42,25 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives {
               }
             }
           }
-        }
+        } ~
+          pathPrefix("policy") {
```

Comment:
i think a path of `/google/resources/{resourceTypeName}/{resourceId}/policies/{accessPolicyName}/sync` would make more sense. it would more closely reflect the structure of our data and also it would be very similar to our existing policy endpoints